README.md

```diff
@@ -3164,17 +3164,46 @@ tweak the implementation to your application's needs, you can follow the steps i
 In order to test Hazelcast C++ client locally, you will need the following:
 
 * Java 8 or newer (for server)
-* Maven
-* cmake
-* OpenSSL
-* Boost
-* thrift
+* Maven 3 or newer (for server)
+* cmake 3.10 or newer
+* OpenSSL 1.0.2 or newer
+* Boost >= 1.71
+* thrift 0.13
```

Comment:
You may use a different version of thrift but this is the one tested with, hence, I am not sure about making such a bold claim.

Reply:
v0.13 is the latest release of Thrift and it was released in Oct 2019. Pinning the version to 0.13 seems to be a pretty safe bet to me. I can include any other version you've tried.